os-datavirt/added/standalone-openshift.xml

```diff
@@ -525,14 +525,14 @@
             <preparedplan-cache infinispan-container="teiid-cache"/>
             <transport name="embedded"/>
             <transport name="odata">
-                <authentication security-domain="teiid-security"/>
+                <authentication security-domain="##ODATA_SECURITY_DOMAIN##"/>
```

Comment:
Is this something we'd want to make generic, e.g. TRANSPORTS=ODATA, ODATA_TRANSPORT_NAME=odata, ODATA_TRANSPORT_SECURITY_DOMAIN=teiid-security.  Obviously, we're limited as to what we might do for existing transports, so maybe another option is to modify teiid-security to handle this?